_layouts/home.html

```diff
@@ -18,4 +18,24 @@
       </li>
     {% endfor %}
   </ul>
+
+  <br>
+  <h3>Number of downloads for the gui version:</h3>
+  <div id="downloads"></div>
+  <script>
+    function createElement(version1, version2) {
+      var tag = document.createElement('dl');
+      var template = '<dl>'
+      +  '<dd>Windows: ' + version1 + '</dd>'
+      +  '<dd>Linux: ' + version2 + '</dd>';
+      tag.innerHTML = template;
+      document.getElementById('downloads').appendChild(tag);
+    }
+    var tags = {};
+    $.ajax({
+      url: "https://api.github.com/repos/mozilla/mozregression/releases"
+    }).done(function(data1) {
+      createElement(data1[0].assets[0].download_count, data1[0].assets[1].download_count);
```

Comment:
This is only the number of download for the latest release - and I'm not sure that 0 will always be windows (and 1 linux), I suspect it depends on the time required to build the release (we are using travis-ci and appveyor for that).

Could you make the change to count the number of downloads for all release ? Also I would prefer to look at the "name" property of an asset (instead of using indexes): if it ends with ".exe", this is for windows, else if it ends with ".tar.gz" this is for linux (please do the two tests as we will add mac one day or the other).